src/discord-rest_request.c

```diff
@@ -171,7 +171,13 @@ _discord_request_info_extract(struct discord_requestor *rqtor,
 {
     ua_info_extract(req->conn, info);
 
+    memset(&req->json, 0, sizeof(struct ccord_szbuf_readonly));
```

Comment:
It would be better to only use that when .start is not set, avoiding doing that all the time.